cla_backend/settings/base.py

```diff
@@ -64,6 +64,9 @@ def env_var_truthy_intention(name):
     "CALLBACK_CREATED_THIRD_PARTY": os.environ.get(
         "GOVUK_NOTIFY_TEMPLATE_CALLBACK_CREATED", "d07bb321-bd1d-4fc1-8b23-80eb0f1e59a1"
     ),
+    "CALLBACK_CAPACITY_THRESHOLD": os.environ.get(
+        "CALLBACK_CAPACITY_THRESHOLD", "867845e8-9052-492e-b891-2dea72fe1220"
```

Comment:
Can we give this a name which fits the naming convention above I.E. `GOVUK_NOTIFY_TEMPLATE_CALLBACK_CAPACITY_ALERT`